src/platform/packages/shared/kbn-zod-helpers/src/deep_strict.ts

```diff
@@ -311,11 +311,11 @@ export function DeepStrict<TSchema extends z.Schema>(schema: TSchema) {
     const excessKeys = difference([...keys.all], [...keys.handled]);
 
     if (excessKeys.length) {
-      context.addIssue({
-        code: ZodIssueCode.unrecognized_keys,
-        keys: excessKeys,
-        message: `Excess keys are not allowed`,
-      });
+      // context.addIssue({
+      //   code: ZodIssueCode.unrecognized_keys,
+      //   keys: excessKeys,
+      //   message: `Excess keys are not allowed`,
+      // });
```

Comment:
The changes in [[Streams 🌊] Improve typing for samples and simulation docs](https://github.com/elastic/kibana/pull/209991#top) should fix this part to which can be uncommented once it's merged.